client/src/components/Navigation.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { GiHamburgerMenu } from "react-icons/gi";
+import { ImCancelCircle } from "react-icons/im";
+import { AiOutlineHome } from "react-icons/ai";
+import { MdOutlineAttachMoney, MdOutlineMoneyOffCsred } from "react-icons/md";
+import { FaRegHandshake } from "react-icons/fa";
+import { motion } from "framer-motion";
+import { Link } from "react-router-dom";
+
+const Container = styled.div`
```

Comment:
What do you think about making a navigation.styles.ts file to hold all of our styled components for navigation? I think it would clean up a lot of this component.